enterprise/frontend/src/embedding-sdk/store/reducer.ts

```diff
@@ -12,10 +12,14 @@ import { createAsyncThunk } from "metabase/lib/redux";
 import { getSessionTokenState } from "./selectors";
 
 const SET_LOGIN_STATUS = "sdk/SET_LOGIN_STATUS";
+const SET_METABASE_CLIENT_URL = "sdk/SET_METABASE_CLIENT_URL";
```

Comment:
Not strictly related to this PR and we can address it later, but do we have any reason to create those consts?
I know they've been "a thing" in the past but now with redux toolkit we could just inline them in the createAction and save a few lines (and layers)